O/OpenSSL/Compat.toml

```diff
@@ -1,10 +1,10 @@
 [1]
-BitFlags = "0.1"
```

Comment:
I'd skip this package because this is independent from https://github.com/JuliaPackaging/Yggdrasil/pull/6312.  You'd need to coordinate with upstream `OpenSSL.jl`.

Reply:
Fixed in 99e778b3655b469ea9a4ef5a93e2e102d7a5c2a9

Reply:
See https://github.com/JuliaWeb/OpenSSL.jl/pull/15 for upstream coordination.